docs/docs/configuration/db.md

```diff
@@ -1,126 +1,151 @@
 
-<details>
-<summary>Result</summary>
+- [Java vulnerability scanning](../coverage/language/java.md).
 
-```
-2019-05-16T12:48:08.703+0900    INFO    Detecting Alpine vulnerabilities...
-
-python:3.4-alpine3.9 (alpine 3.9.2)
-===================================
-Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 0)
-
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-| openssl | CVE-2019-1543    | MEDIUM   | 1.1.1a-r1         | 1.1.1b-r1     | openssl: ChaCha20-Poly1305     |
-|         |                  |          |                   |               | with long nonces               |
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-```
+## Locations
+
+Following are official locations of Trivy databases:
+
+| Registry | Image Address | Link
+| --- | --- | ---
+| GHCR | `ghcr.io/aquasecurity/trivy-db:2` | <https://ghcr.io/aquasecurity/trivy-db>
+| | `ghcr.io/aquasecurity/trivy-java-db` | <https://ghcr.io/aquasecurity/trivy-java-db>
+| | `aquaghcr.io/aquasecurity/ecurity/trivy-checks` | <https://ghcr.io/aquasecurity/trivy-checks>
+| Docker Hub | `aquasec/trivy-db:2` | <https://hub.docker.com/r/aquasec/trivy-db>
+| | `aquasec/trivy-java-db:1` | <https://hub.docker.com/r/aquasec/trivy-java-db>
+| AWS ECR | `public.ecr.aws/aquasecurity/trivy-db:2` | <https://gallery.ecr.aws/aquasecurity/trivy-db>
+| | `public.ecr.aws/aquasecurity/trivy-java-db:1` | <https://gallery.ecr.aws/aquasecurity/trivy-java-db>
+
+ VEX Hub is fetched from VEX Hub GitHub Repository directly: <https://github.com/aquasecurity/vexhub>.
+
+### Automatic fallback
+
+Trivy will attempt to pull images from the official registries in the order specified. In case of failure of pulling a database, Trivy will fall back to the next alternative registry in the order specified.  
+You can specify additional alternative repositories as explained in the [configuring database locations section](#locations).
+
+The Checks Database registry location option does not support fallback through multiple options.
+
+## Connectivity requirements
+
+| database | location | protocol | hosts
+| --- | --- | --- | ---
+| <ul><li>`trivy-db`</li><li>`trivy-java-db`</li><li>`checks-db`</li></ul> | GHCR | [OCI Distribution](https://github.com/opencontainers/distribution-spec) over HTTPS | <ul><li>`ghcr.io`</li><li>`pkg-containers.githubusercontent.com`</li></ul>
+| `vexhub`| GitHub | Git over HTTPS | <ul><li>`api.github.com`</li><li>`codeload.github.com`</li></ul>
+
```

Comment:
fixing table borders

```suggestion
| database                                                         | location | protocol                                                                           | hosts                                                                      |
|------------------------------------------------------------------|----------|------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
| `trivy-db`</li><li>`trivy-java-db`</li><li>`checks-db`</li></ul> | GHCR     | [OCI Distribution](https://github.com/opencontainers/distribution-spec) over HTTPS | <ul><li>`ghcr.io`</li><li>`pkg-containers.githubusercontent.com`</li></ul> |
| `vexhub`                                                         | GitHub   | Git over HTTPS                                                                     | <ul><li>`api.github.com`</li><li>`codeload.github.com`</li></ul>           |


```

Reply:
Should we separate out the three DBs into their own rows?

Reply:
actually I don't like adding extra spaces and dashes in markdown, it makes editing very annoying